source/adapters/level_zero/image.cpp

```diff
@@ -1074,6 +1074,20 @@ UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesMapExternalArrayExp(
   return UR_RESULT_SUCCESS;
 }
 
+UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesMapExternalLinearMemoryExp(
+    ur_context_handle_t hContext, ur_device_handle_t hDevice, uint64_t offset,
+    uint64_t size, ur_exp_interop_mem_handle_t hInteropMem, void **phRetMem) {
+  std::ignore = hContext;
+  std::ignore = hDevice;
+  std::ignore = size;
+  std::ignore = offset;
+  std::ignore = hInteropMem;
+  std::ignore = phRetMem;
+  logger::error(logger::LegacyMessage("[UR][L0] {} function not implemented!"),
```

Comment:
No need for `logger::LegacyMessage` in any new logger uses. It exists only to maintain compatibility with sycl tests that expected an exact log output.

Reply:
Removed.